documentation/functions/domain/SPF_BUILDER.md

```diff
@@ -16,8 +16,8 @@ parameter_types:
   overhead1: string?
   raw: string?
   ttl: Duration?
-  txtMaxSize: string[]
-  parts: number?
+  txtMaxSize: string?
+  parts: string[]?
```

Comment:
`parts` is not documented (further down in this file` as optional, so it's possible that these were just straight swapped and that this should be:
```suggestion
  txtMaxSize: number?
  parts: string[]
```

Reply:
Fixed!